config.template.toml

```diff
@@ -0,0 +1,180 @@
+###################### OpenDevin Configuration Example ######################
+#
+# All settings have default values, so you only need to uncomment and
+# modify what you want to change
+# The fields within each section are sorted in alphabetical order.
+#
+##############################################################################
+
+#################################### Core ####################################
+# General core configurations
+##############################################################################
+[core]
+# API key for E2B
+#e2b_api_key = ""
+
+# Base path for the workspace
+workspace_base = "./workspace"
```

Comment:
Just a thought, this is a little strange for a while: the backend needs it to be an absolute path. This default works only because we get the absolute path if it's relative. I don't know, might as well keep it like this. It's simpler for the user, right?

Reply:
Ah, I also struggled with this before, I prefer the abs path too. However, in the end, I simply commented out the configs like running the `make setup-config` to generate.